### DISCUSSION TITLE
Ignore PHP check

### DISCUSSION THREAD

**Discussion by hongkhanh84** [composer/composer#12100] [Q&A / Support]
I want ignore check Composer detected issues in your platform: Your Composer dependencies require a PHP version ">= 8.2.0".
i have already comment         require __DIR__ . '/platform_check.php';
in autoload_real.php in vendor but not work

**Comment by fredden** [comment]
@hongkhanh84 can you take a step back and explain what you're actually trying to do? It seems that you have a task to complete or a problem to solve, and that you have made certain decisions already which have lead you to this error message. I wonder if there are other decisions which completely avoid this problematic path.

One way to change the PHP requirement would be to change the `require` (and possibly also `require-dev`) sections in `composer.json` and run `composer update`. I don't think modifying things in the `vendor/` directory by hand is supported as Composer "owns" all files therein. (Any changes you make may be thrown away when Composer runs next.)

**Comment by Seldaek** [comment]
You can set `composer config.platform-check false` to disable this entirely (it will apply after you rerun composer install) https://getcomposer.org/doc/06-config.md#platform-check

But yes, it'd be better to fix your requirements, or fix the installed php version. Just ignoring these errors will likely lead to problems down the line.
